src/utils/re_frame.cljs

```diff
@@ -84,3 +84,5 @@
 (def sub (comp deref re-frame/subscribe))
 
 (def dispatch re-frame/dispatch)
+
+(def dispatch-sync re-frame/dispatch-sync)
```

Comment:
could you please elaborate on why sync is needed ?

Reply:
it fine to have it in atom, but in local state, so new atom for new bottom sheet